src/scss/components/card/card.stories.ts

```diff
@@ -41,3 +41,10 @@ export const CardWithMenu: Story = {
     </div>
   `,
 };
+
+export const CardGallery: Story = {
+  render: () =>
+    html`<div class="iati-card-gallery">
+      ${Array(5).fill(Card.render?.call({}))}
```

Comment:
Just curious, why is it Array(5)? Is that just a decision that the card gallery will contain 5 cards?

Reply:
Ah, I was reading way too much into it :laughing: